src/evaluator/from-flake.nix

```diff
@@ -0,0 +1,22 @@
+{ inputs
+, system
+, ...
+}:
+let
+  evaluatorOutputs = import ./default.nix {
+    # TODO: How to compute this on Flakes?
+    makesExecutionId = "123";
```

Comment:
The only way to get a random seed into a flake is to dynamically generate the flake itself.

What is this practically used for? Invalidating some state?

Reply:
Maybe there is a way to expose `/dev/random` / `/dev/urandom` to an impure derivation, though?

I have once ventured into these fields, but not succeeded.

See: https://github.com/NixOS/nixpkgs/blob/610ceb54e34aa95ec0a500d0021a22a55afda416/pkgs/stdenv/generic/make-derivation.nix#L324-L329

Reply:
yeah, we use it in 1 job to invalidate state, we can find another way and drop this feature

Reply:
is that state in any related to the git commit? then, we could peg it to `self.rev`?